config/config.go

```diff
@@ -65,7 +66,15 @@ type AwsConfig struct {
 }
 
 type GcpConfig struct {
-	ProjectId    string `config:"project_id"`
+	// empty for OrganizationAccount
+	ProjectId string `config:"project_id"`
+
+	// empty for SingleAccount
+	OrganizationId string `config:"organization_id"`
+
+	// SingleAccount or OrganizationAccount
+	AccountType string `config:"account_type"`
+
 	GcpClientOpt `config:"credentials"`
 }
```

Comment:
we could remove `account_type` and only rely on the presense of either fields: `project_id` / `organization_id`, but AWS has this field so this is consistent with cloud vendors org support